.chloggen/otelarrow-spanerrors.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Nit: The `note` saying "eliminate error" seems to point to this being a bug fix. Is there a reason it's an enhancement instead?

Reply:
Fixed.